### DISCUSSION TITLE
Help Requested: Test Projects for MDK2 Minifier

### DISCUSSION THREAD

**Discussion by malware-dev** [malware-dev/MDK-SE#316] [Announcements]
Hi guys. I'm now about to slowly start putting together the new minifier for the MDK2 project. In that regard, I need test projects. The larger and more complex, the better. If you use the more advanced region features of the minifier, that's fantastic.

Links to GitHub repos, zip-files uploaded here... If you don't want to share your project with the world, but will allow me access, you can reach out to me on [Keen's Discord](https://discord.com/invite/keenswh) and give me directly.

Thank you!

**Comment by Wicorel** [comment]
I have HUGE, complex projects.

https://github.com/Wicorel/WicoSpaceEngineers/tree/master/Modular/WMNav

https://github.com/Wicorel/WicoSpaceEngineers/tree/master/Modular/WMDock

https://github.com/Wicorel/WicoSpaceEngineers/tree/master/Modular/WMBase

**Comment by InfernoSuperNova** [comment]
I have a monolithic Aimbot/drone script thing:
https://github.com/InfernoSuperNova/spengiescombatscript
